src/Extensions/SwiftypeFileCrawlerExtension.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Ichaber\SSSwiftype\Extensions;
+
+use SilverStripe\Assets\File;
+
+/**
+ * @property File|$this $owner
+ */
```

Comment:
Don't need this, as it's defined in `SwiftTypeCrawlerExtension`.
```suggestion
```

You're using `getOwner()` in this file though, and we know this extension is meant to be applied to `File`, so it might be nice to add an `@method File|$this getOwner()`